src/contracts/interfaces/IAVSDirectory.sol

```diff
@@ -35,6 +58,59 @@ interface IAVSDirectory is ISignatureUtils {
      */
     function deregisterOperatorFromAVS(address operator) external;
 
+	/**
+	 * @notice Called by AVSs to add an operator to an operator set
+	 * 
+	 * @param operator the address of the operator to be added to the operator set
+	 * @param operatorSetID the ID of the operator set
+	 * @param operatorSignature the signature of the operator on their intent to register
+	 * @dev msg.sender is used as the AVS
+	 * @dev operator must not have a deregistration from the operator set
+	 * @dev if this is the first operator set in the AVS that the operator is 
+	 * registering for, a OperatorAVSRegistrationStatusUpdated event is emitted with 
+	 * a REGISTERED status
+	 */
+	function registerOperatorToOperatorSet(address operator, uint32 operatorSetID, ISignatureUtils.SignatureWithSaltAndExpiry memory operatorSignature) external;
```

Comment:
Can we make this an array of OperatorSetIds, update to `uint32[] calldata operatorSetIds`? Otherwise the Operator needs to provide a signature for each operatorSetId they are registering for.
Similarly for deregister as well, take an array of OperatorSetIds

Reply:
cc @0xClandestine

Reply:
done :)